plugins/check_port/conf.php

```diff
@@ -5,3 +5,5 @@
 				// false - disable check_port plugin
 				// "yougetsignal" - use https://www.yougetsignal.com/tools/open-ports/
 				// "portchecker" - use https://portchecker.co/
+
+$preferIpv4 = true;		// prefer ipv4 addresses when checking ports
```

Comment:
If you change this to false, I will merge the pull request before releasing 4.0 stable.

We can discuss enabling this feature by default a later date. I would like to get your contribution added to ruTorrent.

Reply:
Also, can you change the variable names and the comments? We're not preferring IPV4 here. We're looking up the IPV4 address. We need to make this clear, so the client knows what they're enabling.

Reply:
Renamed to `$useIpv4`, or should it be more explicit? Like `$useIpv4Interface` ?